src/Geometry/GMUGeoJSONParser.m

```diff
@@ -106,7 +106,7 @@ - (instancetype)initWithStream:(NSInputStream *)stream {
 
 - (instancetype)initWithURL:(NSURL *)url {
   if (self = [super init]) {
-    _stream = [[NSInputStream alloc] initWithURL:url];
+    _data = [[NSData alloc] initWithContentsOfURL:url];
```

Comment:
Maybe OK for now but I'd like to move this expensive operation into the parse method later.